composer.json

```diff
@@ -38,14 +38,15 @@
     },
     "require-dev": {
         "doctrine/migrations": "^2.0 || ^3.0",
+        "elasticsearch/elasticsearch": "^7.17 || ^8.5",
```

Comment:
I don't think we should use the `elasticsearch/elasticsearch` package in order to query its status: elasticsearch can/should communicate with its REST-ish API

Reply:
That makes my life/PR a lot easier